### DISCUSSION TITLE
Finetuning Whisper ASR Model for Translation Tasks

### DISCUSSION THREAD

**Discussion by kikozi2000** [openai/whisper#1343] [Q&A]
Hi there,

I'm interested in finetuning the Whisper ASR model to perform translation tasks. Has anyone here tried this before? If so, could you provide some guidance on the required dataset format and the process of training the model for this purpose? Any insights or experiences shared would be greatly appreciated.

Thanks in advance!

**Comment by phineas-pta** [comment]
i haven't done any fine-tuning, but i think you can just follow the official guide, when you load the tokenizer, you change to `task="translate"`

**Comment by WassayS** [comment]
@kikozi2000 Hey, did you try fine tuning it for translation tasks?
If yes, please share your experience like did it work

**Comment by ExtReMLapin** [comment]
At this point, just use NLLB-200 for translation, don't use whisper
